JASP-Engine/JASP/R/LScommon.R

```diff
@@ -160,6 +160,9 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
       tempArrow       <- allArrows[i,]
       tempArrow$yEnd <- tempArrow$yEnd * .scalingSpikes(allLines, allArrows)
       
+      if (tempArrow$yEnd == 0)
+        tempArrow$yEnd <- 1e-5
```

Comment:
if plotting goes wrong for exactly 0, leave as it is. If plotting would also go wrong for 1e-300, I'd change this to:
```suggestion
      if (tempArrow$yEnd < .Machine$double.eps)
        tempArrow$yEnd <- 1e-5
```
The same goes for the other checks for 0

Reply:
it's probably only zero, but I agree that this is safer. Changed